### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: src/main/groovy/de/laser/reporting/myInstitution/LicenseQuery.groovy [hunk 1]
```diff
@@ -22,6 +22,16 @@ class LicenseQuery extends BaseQuery {
 
         if (! idList) {
         }
+        else if ( suffix in ['*']) {
+
+            handleGenericAllQuery(
+                    params.query,
+                    'select l.reference, l.reference, count(l.reference) from License l where l.id in (:idList) group by l.reference',
```
Comment (croqueGrec09) [inline src/main/groovy/de/laser/reporting/myInstitution/LicenseQuery.groovy:29]: duplicate l.reference - intended?

File: src/main/groovy/de/laser/reporting/myInstitution/OrganisationQuery.groovy [hunk 2]
```diff
@@ -30,6 +30,16 @@ class OrganisationQuery extends BaseQuery {
 
         if (! idList) {
         }
+        else if ( suffix in ['*']) {
+
+            handleGenericAllQuery(
+                    params.query,
+                    'select o.name, o.name, count(o.name) from Org o where o.id in (:idList) group by o.name',
```
Comment (croqueGrec09) [inline src/main/groovy/de/laser/reporting/myInstitution/OrganisationQuery.groovy:37]: duplicate o.name - intended?

File: src/main/groovy/de/laser/reporting/myInstitution/SubscriptionQuery.groovy [hunk 3]
```diff
@@ -27,6 +27,16 @@ class SubscriptionQuery extends BaseQuery {
 
         if (! idList) {
         }
+        else if ( suffix in ['*']) {
+
+            handleGenericAllQuery(
+                    params.query,
+                    'select s.name, s.name, count(s.name) from Subscription s where s.id in (:idList) group by s.name',
```
Comment (croqueGrec09) [inline src/main/groovy/de/laser/reporting/myInstitution/SubscriptionQuery.groovy:34]: duplicate s.name - intended?

**Review by croqueGrec09 — COMMENTED** [review]
remarks

File: src/main/groovy/de/laser/reporting/myInstitution/LicenseQuery.groovy (hunk 1 above)
Comment (klodav) [reply src/main/groovy/de/laser/reporting/myInstitution/LicenseQuery.groovy:29]: yes

File: src/main/groovy/de/laser/reporting/myInstitution/OrganisationQuery.groovy (hunk 2 above)
Comment (klodav) [reply src/main/groovy/de/laser/reporting/myInstitution/OrganisationQuery.groovy:37]: yes

File: src/main/groovy/de/laser/reporting/myInstitution/SubscriptionQuery.groovy (hunk 3 above)
Comment (klodav) [reply src/main/groovy/de/laser/reporting/myInstitution/SubscriptionQuery.groovy:34]: yes

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**